src/components/fullScreenNotice.tsx

```diff
@@ -4,7 +4,7 @@ import { colors, Typography } from '@material-ui/core'
 import { Button } from './button'
 
 interface Props {
-  onClick?: () => void
+  clickHandler?: () => void
```

Comment:
Let's add `color` to this props, this will replace what is used on line 38. We default to `green` to keep backward compatibility. 

```
import { Props as ButtonProps } from '../components/button'

interface NoticeProps {
  color?: ButtonProps['color']
}
```

Reply:
kind of did this but let's talk about it.